### DISCUSSION TITLE
(如果你害怕蜘蛛 那么别打开该贴)白额高脚蛛可爱吗

### DISCUSSION THREAD

**Discussion by KELERSAMA** [share121/inter-knot#5265] [公告]
蟑螂又油又吓人，恶心，蟑螂坏 
白额高脚蛛吃蟑螂，怕人，白额高脚蛛好

![image](https://github.com/user-attachments/assets/e6b57a4d-6c21-4668-8669-0db09b03734a)

![image](https://github.com/user-attachments/assets/d543c787-3279-42f9-bc90-8defbd143277)

**Comment by Uzawa-Reisa** [comment]
点进来首页一个蜘蛛。。。蜘蛛恐惧症直接去世了。。这APP还没有屏蔽帖子的功能

**Reply by KELERSAMA** [reply]
实在抱歉
改了
<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >

**Comment by Yizakl** [comment]
![IMG_0125](https://github.com/user-attachments/assets/c851d1a0-9ea8-462a-bbd6-bd99fde28398)
喵呜
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Yizakl** [comment]
huohuo是谁，我是Yizakl（不是

**Comment by manyshoes** [comment]
大学有个广东朋友喜欢养蜘蛛，我看照片都能掉半条命的人理解不了😭

**Reply by KELERSAMA** [reply]
可以治蟑螂的就是好人 😭

**Reply by manyshoes** [reply]
<img src="https://img-static.mihoyo.com/communityweb/upload/fbf3886200751650904a8dc762e799f3.gif" alt="崩坏娘可怜" width="75" >幸亏住在内地，从小到大没见过蟑螂

**Reply by KELERSAMA** [reply]
> 地址在哪里 我委托天马速运给你包邮寄一公一母两只油光满盈 触须可以直接拔下来榨油 拳头大小的飞天蟑螂 就是不知道母的那只踩死了会不会爆蟑螂宝宝
 <img alt="阿姬倒地了03" width="75" src="https://camo.githubusercontent.com/18fe57e461b314be4d55c937287ddbbd44044e5d69cb7f2e730f73cc3c55bb79/68747470733a2f2f6262732d7374617469632e6d69796f757368652e636f6d2f7374617469632f323032332f30352f32332f61326661383035613061373832636462386162343464646166386435333433645f343934313632383539333532383530373733342e706e67">

**Reply by manyshoes** [reply]
比十亿个杀人魔更恐怖，更可怕呀<img src="https://bbs-static.miyoushe.com/static/2024/07/31/ed2b65b6899fc16528b1d10adfcb9a38_8705924363701631267.png" alt="云璃 白眼" width="75" >

**Comment by hot-ice-seele** [comment]
可爱
